software-engineering-intern-python/README.md

```diff
@@ -0,0 +1,41 @@
+# Software engineering intern Recrutiment Task
+
+Thank you for taking the time to solve this assignment! We're excited to see what you will build and hope you'll have some fun along the way!
+
+If you didn't have a chance before, have a look at our [website](https://gyfted.me/) to get a deeper understanding of what we're building at Gyfted.
+
+
+## Introduction
+Your task is to implement a small script, which will allow our psychometricians to see how quizzes are built in the system (what questions are in the quiz etc).
+
+### Let us introduce one word, which will make the task easier to understand.
+Trait - psychological measure, may be directly or indirectly calculated from the questions.
+E.g. to calculate "a big trait" such as 'openness to experience' we need 'smaller' traits such as 'positive thinking' or 'sociability'. 
+
+To be precise let's call 'opennes to experience' - (parent) trait, and 'positivte thinking' & 'sociability' subtraits.
+
+In each subtrait object we hold the information about it's parent trait. From the example above 'positivte thinking' will hold id of the 'openness to experiences' trait in parentTraitId.
+ 
+
+Each quiz contains a few parent traits and subtraits.
```

Comment:
it's unclear how traits are connected to quizzes. Does quiz object hold trait ids?